app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -298,14 +298,23 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
                           ? 'CURRENT'
                           : `WEEK ${index + 1}`}
                       </span>
-                      <Image
-                        className="league-entry-logo"
-                        width={64}
-                        height={64}
-                        data-testid="league-history-logo"
-                        src={logoURL}
-                        alt="teamLogo"
-                      />
+                      {logoURL ? (
+                        <Image
+                          className="league-entry-logo"
+                          width={64}
+                          height={64}
+                          data-testid="league-history-logo"
+                          src={logoURL}
+                          alt="teamLogo"
+                        />
+                      ) : (
+                        <span
+                          className="text-xs h-16 w-16 text-primary pt-6 text-center"
```

Comment:
I'm worried about the a11y of the color and the small text size, did you check this?

Reply:
@ryandotfurrer I did not check this.  hmmmmm @shashilo what do you think?

Reply:
Fine for now. A11Y audit can come after.